helmfile.d/config/registrar-test/registrar-challenger-values.yaml.gotmpl

```diff
@@ -2,4 +2,16 @@ config:
   watcher_url: "ws://test-registrar-watcher:3001"
   matrix_homeserver: {{ env "TEST_MATRIX_HOMESERVER" }}
   matrix_username: {{ env "TEST_MATRIX_USER" }}
-  matrix_password: {{ env "TEST_MATRIX_PASSWORD" }} 
\ No newline at end of file
+  matrix_password: {{ env "TEST_MATRIX_PASSWORD" }}
```

Comment:
the env are the sensitive ones:

@lamafab Do you agree that `twitter_screen_name`, `google_scope`, `email_server` are not sensive ?

Reply:
Yep, it's okay if those are public.